app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  validates :user_id, presence: true
```

Comment:
- [ ] `belongs_to` association already adds presence validation to `user_id`. So, we don't need it.

Reply:
Thanks for the heads-up on that!